app/jobs/scheduled/ignored_users_summary.rb

```diff
@@ -0,0 +1,45 @@
+module Jobs
+  class IgnoredUsersSummary < Jobs::Scheduled
+    every 1.day
+
+    def execute(args)
+      return unless SiteSetting.ignore_user_enabled
+
+      params = {
+        threshold: SiteSetting.ignored_users_count_message_threshold,
+        gap_days: SiteSetting.ignored_users_message_gap_days
+      }
+      user_ids = DB.query_single(<<~SQL, params)
+        SELECT ig.ignored_user_id AS user_id
+        FROM ignored_users AS ig
+        WHERE NOT EXISTS (SELECT 1
+                          FROM post_custom_fields as pcf
+                          WHERE pcf.name = 'summary_sent_for_ignored_user'
+                            AND pcf.value = user_id::text
+                            AND pcf.created_at < CURRENT_TIMESTAMP - ':gap_days DAYS'::INTERVAL
```

Comment:
If it is 1 day, I believe this Postgres syntax: `1 DAYS` is still okay?

Reply:
Only way to know for sure is to test it ;)